src/components/ReviewRoutingSlip/ReviewRoutingSlipDetails.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <v-row no-gutters class="mb-16" v-if="routingSlipDetails">
+    <v-col class="col-12 col-sm-10 ">
```

Comment:
col-sm-10 needed?

Reply:
good catch. i ll have just the col-12 grid alone